components/description-list/description-list-wrapper.js

```diff
@@ -41,6 +41,12 @@ class DescriptionListWrapper extends LitElement {
 			 * @type {number}
 			 */
 			breakpoint: { type: Number, reflect: true },
+			/**
+			 * Force the component to always use a stacked layout
+			 * This will override the breakpoint attribute
+			 * @type {boolean}
+			 */
+			forceStacked: { type: Boolean, reflect: true, attribute: 'force-stacked' },
```

Comment:
We have been duplicating the attribute list in the README, so probably worth adding this there:
https://github.com/BrightspaceUI/core/tree/main/components/description-list#properties

Reply:
Yeah, still need to update documentation. Was gonna do that once the technical things have been worked out in case naming, etc changes 😄

Reply:
Cool -- what you have works for me!